### DISCUSSION TITLE
Expo Phone Auth Silent APNs notifications

### DISCUSSION THREAD

**Discussion by sirusbaladi** [invertase/react-native-firebase#7085] [Q&A]
Hi All,

I'm Using Expo managed workflow with EAS Build. 

I followed the instructions to install react-native-firebase and added the plugin for using phone auth but I'm encountering two issues. 

1) When I go to the page to verify the code, it always opens a captcha and it never uses the silent notifications. Is there a way to enable them?
2) The recaptcha has 7/8 seconds latency. Any way to speed that up?

**Comment by sunnyysetia** [comment]
I am having the same issue. Please let me know if you have figured this out?

**Reply by sirusbaladi** [reply]
yes register to receive notifications:
https://docs.expo.dev/versions/latest/sdk/notifications/#notification-events-listeners

**Reply by appsgenie** [reply]
do yall know if I need to set this in app.json? ios->infoPlist->UIBackgroundModes->["remote-notification"] for the silent notifications to work?
also what is a custom url scheme?

**Comment by appsgenie** [comment]
also wondering how do you customize the URL where reCAPTCHA verification happens.. was able to do it with the JS SDK...
